gnome-image-installer/util/gis-unattended-config.c

```diff
@@ -162,12 +175,42 @@ static gboolean
 gis_unattended_config_populate_fields (GisUnattendedConfig *self,
                                        GError **error)
 {
+  g_auto(GStrv) groups = g_key_file_get_groups (self->key_file, NULL);
+  gchar **group;
```

Comment:
`const gchar **`?

Reply:
```
  gelf ~
% cat /tmp/foo.c
#define NULL ((void *) 0)

int main()
{
  char *arr[2] = { "abc", NULL };
  char **pp = &arr[0];
  const char **iter = pp;
  return iter[0] == NULL;
}
  gelf ~
% gcc -Wall -Wextra -o /tmp/foo /tmp/foo.c
/tmp/foo.c: In function ‘main’:
/tmp/foo.c:7:23: warning: initialization from incompatible pointer type [-Wincompatible-pointer-types]
   const char **iter = pp;
```

I don't think adding a cast would make this code more legible.